challenge-05/challenge-05.js

```diff
@@ -47,7 +56,33 @@ propriedades:
 - Se o parâmetro não for passado, a função deve retornar o objeto com todos
 os livros.
 */
-// ?
+function book(bookname){
+	var table = {
+		"bookname1": {
+			quantidadePaginas: 100,
+			autor: "Fernando Daciuk",
+			editora: "Uma ai"
+		},
+		"bookname2": {
+			quantidadePaginas: 200,
+			autor: "Mateus Jabour",
+			editora: "Aquela"
+		},
+		"bookname3": {
+			quantidadePaginas: 10,
+			autor: "Fulano",
+			editora: "Outra la"
+		}
+	}
+
+		if (bookname !== undefined){
```

Comment:
Aqui você está fazendo o contrário: se o argumento **FOR DIFERENTE DE `undefined`**, então, retorne o objeto todo. Ou seja, para o argumento ser diferente de `undefined`, ele precisa ter sido passado, correto?